response.go

```diff
@@ -3,12 +3,40 @@ package algnhsa
 import (
 	"encoding/base64"
 	"net/http/httptest"
+	"unicode"
 
 	"github.com/aws/aws-lambda-go/events"
 )
 
 const acceptAllContentType = "*/*"
 
+// binaryCase ported from https://github.com/Gi60s/binary-case/blob/b100ba0d63075c28485fd1724d94746f74742107/index.js#L86
+func binaryCase(s string, n int) string {
+	inp := []rune(s)
+	var res []rune
+
+	for i, c := range inp {
+		if n == 0 {
```

Comment:
Does this condition has to be here? It seems to me the code is doing the same thing without the condition. Please correct me if I'm wrong.